workflows/cloudify_system_workflows/deployment_environment.py

```diff
@@ -23,6 +23,23 @@
 from cloudify.workflows import workflow_context
 
 
+def _should_create_policy_engine_core(policy_configuration):
+    """Examine the policy_configuration and decide to start a riemann core
+    """
+    groups = policy_configuration['groups']
+    policy_types = policy_configuration['policy_types']
+
+    if not groups or not policy_types:
+        return False
+
+    for group in groups.values():
+        for policy_config in group['policies'].values():
```

Comment:
a group may have no policies defined (not true yet, but will be true soon) so do something like
`group.get('policies', {})` or similar